### DISCUSSION TITLE
General API Questions

### DISCUSSION THREAD

**Discussion by b2bsun** [Sellbrite/Sellbrite-API#48] [Q&A]
Hi I am completely new to API and had some general questions regarding Sellbrite API.

Once we connect with a website through API, will that website show up on our listing dashboard the way other sales channels do? 
Also on the product section, will this api connected website show up on the filter "Listed on"?

Are we able to turn on inventory sync but have price sync on for this api connected website the way we are able to do so for other sales channels?

To upload new listings, would it work the same way as it does for other channels or will we have to do it through a csv table?

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

You won't be able to connect a sales channel via the Sellbrite API or create listings. The API is used for passing inventory updates, creating products, and fulfilling shipments that have been imported into your Sellbrite account. If you have inventory updates from a third-party site you can use the API to pass inventory updates onto your products in Sellbrite.
